packages/core/contracts/Escrow.sol

```diff
@@ -213,11 +217,22 @@ contract Escrow is IEscrow {
 
         bulkPaid = true;
         balance = getBalance();
-        if (status == EscrowStatuses.Pending) {
-            status = EscrowStatuses.Partial;
-        }
-        if (balance == 0 && status == EscrowStatuses.Partial) {
-            status = EscrowStatuses.Paid;
+        if (bulkPaid) {
+            if (status == EscrowStatuses.Pending) {
+                status = EscrowStatuses.Partial;
+                remainingFortunes = remainingFortunes.sub(_recipients.length);
+            }
+            if (
+                balance > 0 &&
+                status == EscrowStatuses.Partial &&
+                remainingFortunes == 0
+            ) {
+                _safeTransfer(canceler, balance);
```

Comment:
this should be launcher

Reply:
Launcher address contains the escrow factory address and canceller is the owner of the escrow